target/scala-2.11/scala-space-launcher.js

```diff
@@ -0,0 +1,2 @@
+((typeof global === "object" && global &&
+         global["Object"] === Object) ? global : this)["scalaspace"]["ScalaSpace"]().main();
```

Comment:
I agree, but unfortunately I don't know how to automatically move the compiled scripts into the ```js``` directory with SBT. I'm not taking copying them manually in consideration as I'd prefer the process to be automatic and even though I'm not able to do so, I'm pretty sure there's a way. I'll gladly improve this pull request, can you help me with this? :)